code_schemes/rqas/aik/aik_incidents_of_polarisation.json

```diff
@@ -13,6 +13,14 @@
             "VisibleInCoda": true,
             "Shortcut": "e"
         },
+        {
+            "CodeID": "code-7021e633",
+            "CodeType": "Normal",
+            "DisplayText": "yes",
```

Comment:
should this be 'yes, no location'?

Reply:
No, there are two questions for this dataset. This theme is for those who answered the first question only.